src/providers/provider.ts

```diff
@@ -13,6 +13,10 @@ export type ProviderConfig = {
    * Debug flag to test some codepaths
    */
   debugRouting?: boolean;
+  /**
+   * Flag for token properties provider to enable fetching fee-on-transfer tokens.
+   */
+  enableFeeOnTransferFeeFetching?: boolean;
```

Comment:
just as an fyi for me, but who will be setting this on the provider?

Reply:
This was discovered yesterday, where we have to expose a feature flag for the web folks when they integrate. Because they hardcode FOT tax into the quote amount, without the feature flag, once we upgrade the routing-api to include the sor FOT taxation, interface will immediately start double FOT taxation.

More context was discussed [here](https://uniswapteam.slack.com/archives/C05MZUM1E69/p1694128406219509?thread_ts=1694122431.474749&cid=C05MZUM1E69).

This means we need to expose the boolean flag in unifed-routing-api and routing-api as well, but they should be quick to implement.